beacon_node/lighthouse_network/src/rpc/codec.rs

```diff
@@ -583,9 +585,21 @@ fn handle_rpc_request<E: EthSpec>(
                 )?,
             }),
         ))),
-        SupportedProtocol::BlobsByRangeV1 => Ok(Some(RequestType::BlobsByRange(
-            BlobsByRangeRequest::from_ssz_bytes(decoded_buffer)?,
-        ))),
+        SupportedProtocol::BlobsByRangeV1 => {
+            let req = BlobsByRangeRequest::from_ssz_bytes(decoded_buffer)?;
+            // TODO(pawan): change this to max_blobs_per_rpc_request in the alpha10 PR
+            let max_blobs = spec.max_blobs_per_block_by_fork(current_fork);
```

Comment:
Should we multiply this by `req.count`? Otherwise most range requests might be rejected?

Reply:
Ohh yeah good catch, I have a `max_blobs_requests_by_fork ` function in the alpha 10 specs PR that i was planning to use for this, but this will kill us until that gets merged

Reply:
It still looks wrong to me. Shouldn't it be:

```rust
if req.count * max_blobs_per_block > max_request_blob_sidecars {
   // error
}
```